B/BandwidthBenchmark/Compat.toml

```diff
@@ -0,0 +1,6 @@
+[0]
+DataFrames = "1"
+PrettyTables = "1"
```

Comment:
Not a blocker for registration but note that [PrettyTables](https://github.com/ronisbr/PrettyTables.jl) has version 2, which is required by DataFrames 1.4 https://github.com/JuliaRegistries/General/blob/6b93f2390858bbb712b398968b9f51d0b3ab0a5c/D/DataFrames/Compat.toml#L161-L163 so this packages holds DataFrames back.

[noblock]

Reply:
Related, you may want to re-enable [CompatHelper](https://github.com/JuliaPerf/BandwidthBenchmark.jl/actions/workflows/CompatHelper.yml) in your repository

[noblock]

Reply:
Ah, yeah, thanks for the pointer. I have to tag a quick 0.1.1 release anyways in which I'll also fix this. (If I make changes now, will the 3 day period start over again?)

Reply:
If you don't change the version number and trigger registration again, this PR is updated.  Waiting time is always from when the PR was originally opened. 

Also, you may want to use `[noblock]` not to block automatic registration.